platform/windows/build_debug.ps1

```diff
@@ -0,0 +1,13 @@
+Write-Host "Configure build"
+
+$InstallPrefix = Join-Path (Get-Location) bin\windows\debug
+
+cmake -B build -S . -DUSE_VCPKG=1 -DCUSTOM_CMAKE=".\platform\windows\Custom-vs.cmake" -DCMAKE_INSTALL_PREFIX:PATH=$InstallPrefix
+
+Write-Host "Build Csound"
+
+cmake --build build --config Debug
+
+Write-Host "Install Csound"
+
+cmake --build build --target install --config Debug
```

Comment:
Not entirely sure this is the best way to solve this. It only really helps the first time you clone the repo. I imagine if devs are changing code, they don't want to regenerate the whole project but just run the build command and let cmake pick up any changes internally. Maybe this is better solved with Cmake presets? I thought about adding that recently.

Reply:
I created these mainly for reproducing the github windows build.  I had an error with another pull request, but I couldn't see the error in the github logs.  I'm open to updating these to improve them.